src/main/scala/org/broadinstitute/dsde/workbench/sam/config/TermsOfServiceConfig.scala

```diff
@@ -3,8 +3,9 @@ package org.broadinstitute.dsde.workbench.sam.config
 /**
   * Terms of Service configuration.
   * @param enabled Set to false to disable Terms of Service enforcement
+  * @param enableGracePeriod Set to true if the grace period for ToS acceptance is active
```

Comment:
oh here's the definition. Same question as before naming.

Reply:
Ok, I think this is an 'action' variable, and is in line with other configs variables.

Reply:
I renamed it to `isGracePeriodEnabled` per your suggestion. I defer strongly to the reviewer for this type of feedback, since the reviewer is more likely to know whether or not something is readable. I chose the name so I'm biased :)